src/transformers/image_utils.py

```diff
@@ -284,6 +286,10 @@ def load_image(image: Union[str, "PIL.Image.Image"], timeout: Optional[float] =
             # We need to actually check for a real protocol, otherwise it's impossible to use a local file
             # like http_huggingface_co.png
             image = PIL.Image.open(requests.get(image, stream=True, timeout=timeout).raw)
+        elif image.startswith("data:image/"):
+            # Try to load as base64
+            b64 = base64.b64decode(image.split(",")[1])
+            image = PIL.Image.open(BytesIO(b64))
```

Comment:
This assumes that the path is a data URI and it's base64 encoded. The ValueError raised below should also be updated to reflect the expected types. 

I'd suggest doing something like this: 

```
        elif os.path.isfile(image):
            image = PIL.Image.open(image)
        else:
            if image.startswith("data:image/"):
                image = image.split(",")[1]

            try:
                b64_image = base64.b64decode(image, validate=True)
                image = PIL.Image.open(BytesIO(b64_image))

            except Exception as e:
                raise ValueError(
	                f"Incorrect path, string or url. URLs must start with `http://` or `https://`, data a valid base64 string or a valid path. Got {image}. Failed with {e}."
	            )
```